mautic/mautic
# [UX] Quick help

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

this pr introduces a new quick help icon that, when clicked, displays a dropdown menu with useful information for mautic users. the dropdown will include shortcuts to common features:
- Keyboard shortcuts
- Search commands modal

![image](https://github.com/mautic/mautic/assets/116097999/9b7ff8be-6360-4649-bbf3-8a81037c3a30)

this feature is designed to make it easier for users to discover and access important functionality within mautic. when providing a centralized location for these helpful items, users can quickly find the information they need without having to navigate through docs. 

It will allow us to add other things in the future, such as a “Provide feedback” link (if relevant), leave a slot for stakeholders to add a “contact support” or anything that helps the user with the platform or our stakeholders’ needs


Related PRs:
- #13939
- #13942 






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts > Click on the icon on the navbar > Click on both
